yearn/contracts/ConvexStrategy.sol

```diff
@@ -237,24 +267,59 @@ contract ConvexStrategy is BaseStrategy {
         }
     }
 
-    /// @notice Sets the portion of CRV tokens which should be locked in
-    ///         the Curve vote escrow to gain CRV boost.
+    /// @notice Begins the update of the threshold determining the portion of
+    ///         CRV tokens which should be locked in the Curve vote escrow to
+    ///         gain CRV boost.
     /// @dev Can be called only by the strategist and governance.
-    /// @param _keepCRV Portion as counter of a fraction denominated by the
+    /// @param _newKeepCRV Portion as counter of a fraction denominated by the
     ///        DENOMINATOR constant.
-    function setKeepCRV(uint256 _keepCRV) external onlyAuthorized {
-        require(_keepCRV <= DENOMINATOR, "Max value is 10000");
-        keepCRV = _keepCRV;
+    function beginKeepCRVUpdate(uint256 _newKeepCRV) external onlyAuthorized {
+        require(_newKeepCRV <= DENOMINATOR, "Max value is 10000");
+        newKeepCRV = _newKeepCRV;
+        keepCRVChangeInitiated = block.timestamp;
+        emit KeepCRVUpdateStarted(_newKeepCRV, block.timestamp);
     }
 
-    /// @notice Sets the extra reward swap threshold.
+    /// @notice Finalizes the keep CRV threshold update process.
+    /// @dev Can be called only by the strategist and governance, after the the
```

Comment:
Double `the`

Reply:
https://github.com/keep-network/tbtc-v2/pull/48/commits/e8ffe58a855cd49371e1341f9ed94e447d9adc83